app/scripts/metamask-controller.js

```diff
@@ -1876,15 +1887,18 @@ export default class MetamaskController extends EventEmitter {
 
           return { gasLimit, simulationFails };
         },
+        messenger: swapsControllerMessenger,
         provider: this.provider,
         getProviderConfig: () => this.networkController.state.providerConfig,
         getTokenRatesState: () => this.tokenRatesController.state,
         getCurrentChainId: () =>
           this.networkController.state.providerConfig.chainId,
+        // TODO: Remove once GasFeeController exports this action type
         getEIP1559GasFeeEstimates:
           this.gasFeeController.fetchGasFeeEstimates.bind(
             this.gasFeeController,
           ),
+        // TODO: Remove once TransactionController exports this action type
         getLayer1GasFee: this.txController.getLayer1GasFee.bind(
           this.txController,
         ),
```

Comment:
In lines 1905-1906, `getNetworkClientId` should be removed as it's not defined in `SwapsControllerOptions`, nor is it called anywhere in `SwapsController`.

Reply:
You are right here! Old code that shouldnt be there anymore